osquery/tables/applications/browser_utils.cpp

```diff
@@ -71,6 +71,32 @@ Status getChromeProfileName(std::string& name, const fs::path& path) {
 }
 } // namespace
 
+const std::string genPermissions(const std::string& permissionTypeKey,
+                                 pt::ptree tree) {
```

Comment:
Does this work as a const reference? Eg., const pt::ptree& tree

Reply:
It does. Should I change it to that?

Reply:
yes please! Passing the `tree` by reference will prevent an unneeded copy of the underlying data.